HowTos/Troubleshoot_Logs.rst

```diff
@@ -6,9 +6,37 @@
 Logs
 ###################################
 
+Upload tracelog
+---------------
 
+On the controller console left side menu, click Troubleshoot, click Logs and select a gateway at Upload Tracelog. The controller and gateway tracelog will be uploaded to Aviatrix. The Aviatrix support team will be alerted. If no gateway is selected, only the controller log is uploaded.
 
- *  `Upload tracelog. <http://docs.aviatrix.com/HowTos/troubleshooting.html>`__
+Please refer to `Troubleshoot <http://docs.aviatrix.com/HowTos/troubleshooting.html>`__ for troubleshooting detail.
+
+Display Aviatrix Command Log
+----------------------------
+
+DISPLAY
+~~~~~~~
+
+This feature enables users to view Aviatrix Command Log on GUI.
+
+DISPLAY AUDIT
+~~~~~~~~~~~~~
+
+This feature enables users to view Aviatrix Audit log on GUI.
+
+DOWNLOAD AUDIT
+~~~~~~~~~~~~~~
+
+This feature enables users to download Aviatrix Audit log to local.
```

Comment:
download Aviatrix Audit log should be good. Would be good to define what an Audit log is